test/e2e/network/service.go

```diff
@@ -1736,26 +1736,60 @@ var _ = SIGDescribe("Services", func() {
 		}
 	})
 
-	ginkgo.It("should have session affinity work for service with type clusterIP", func() {
-		svc := getServeHostnameService("affinity-clusterip")
+	/*
+		Release: v1.16
+		Testname: Service, ClusterIP type, session affinity ON
+		Description: Create a service of type "ClusterIP". Service's sessionAffinity is set to "ClientIP". Service creation MUST be successful by assigning "ClusterIP" to the service.
+		Create a Replication Controller to ensure that 3 pods are running and are targeted by the service to serve hostname of the pod when requests are sent to the service.
+		Create another pod to make requests to the service. Service MUST serve the hostname from the same pod of the replica for all consecutive requests.
+		Service MUST be reachable over serviceName and the ClusterIP on servicePort.
+	*/
+	framework.ConformanceIt("should have session affinity work for service with type clusterIP", func() {
+		svc := getServeHostnameService("service")
 		svc.Spec.Type = v1.ServiceTypeClusterIP
 		execAffinityTestForNonLBService(f, cs, svc)
 	})
 
-	ginkgo.It("should be able to switch session affinity for service with type clusterIP", func() {
-		svc := getServeHostnameService("affinity-clusterip-transition")
+	/*
+		Release: v1.16
+		Testname: Service, ClusterIP type, session affinity OFF
+		Description: Create a service of type "ClusterIP". Service's sessionAffinity is set to "ClientIP". Service creation MUST be successful by assigning "ClusterIP" to the service.
+		Create a Replication Controller to ensure that 3 pods are running and are targeted by the service to serve hostname of the pod when requests are sent to the service.
+		Create another pod to make requests to the service. Update the service's sessionAffinity to "None". Service update MUST be successful. When a requests are made to the service, it MUST be able serve the hostname from any pod of the replica.
+		When service's sessionAffinily is updated back to "ClientIP", service MUST serve the hostname from the same pod of the replica for all consecutive requests.
+		Service MUST be reachable over serviceName and the ClusterIP on servicePort.
+	*/
+	framework.ConformanceIt("should be able to switch session affinity for service with type clusterIP", func() {
+		svc := getServeHostnameService("service")
```

Comment:
Was there any particular reason for renaming all of these to `service` for each case? I think it was probably useful for debugging to have the different service names

Reply:
Done.